File: WriteFile.c
#include<stdio.h>
#include<fcntl.h>
#include<unistd.h>
#include<string.h>

int main()
{
    int fd = 0;
    char Arr[] = "Rushikesh Bhoi";
    int Ret = 0;

    fd = open("Rushi.txt",O_RDWR);

    Ret = write(fd,Arr,strlen(Arr)); // (Kashat Lihayach, Kay Lihaycha, Kiti Lihaycha)

    printf("%d bytes gets written in the files\n",Ret);

    close(fd);

    return 0;
}
// O_RDONLY : Read
// O_WRONLY : Write
// O_RDWR : Read + Write

File: WriteFile.java
import java.io.*;

class WriteFile
{
    public static void main(String Args[])
    {
        try
        {
           String Data = "Rushikesh Bhoi";
           byte Arr[] = Data.getBytes();

           FileOutputStream fobj = new FileOutputStream("Rushikesh.txt");
           fobj.write(Arr);
        }

        catch(Exception obj)
        {}
    }
}